keps/sig-scheduling/3280-guarantee-pdb-when-preemption-happens/kep.yaml

```diff
@@ -0,0 +1,45 @@
+title: Guarantee PodDisruptionBudget When Preemption Happens
+kep-number: 3280
+authors:
+  - "@denkensk"
+owning-sig: sig-scheduling
+participating-sigs:
+  - sig-auth
```

Comment:
Sill missing sig-apps :wink:

Reply:
We don't need to approve, since it eventually isn't touching our APIs, but it would be good as a participating sig, since it touches sig-apps bits, at least partially.

Reply:
Added. Thanks so much for you review ^_^